### PR TITLE
Ky day 2

### PR DESCRIPTION
**Pull request by KianYang-Lee**
* Add 3 new files which focus on AR model, MA model and stationarity and differencing techniques respectively.
* Revised original ARIMA file by creating a new file and import relevant codes into it
* Removed original ARIMA file
* Add exercise and included in exercise folder

### CODE REVIEW

File: notebook/Solution/02 - Autoregressive Integrated Moving Average (ARIMA).ipynb [hunk 1]
```diff
@@ -0,0 +1,1776 @@
...
...sEqxr9Bcl7kOxJMFYasI+pmNhMgnWNpbxZqsWMfPGiYHcJweFYi6ixF1RkJ5gnD+91cOHLlo0wrWRTb2fjGvmeF94ytUPNyzwc6BjiGiicPZolpN6stN8/9TT+dc1aPc60RemfTDGhgXlUx5LoZhplLgrCpJNHjpvoX/E8Wx2ZrENLsJxrQl0MT7zQmxoLCeVkeydoGpjTtynaLmD5ndPZSQPv9qi3VtZ7oo5jBJ3RUGyUSOnNqVw2bSImbeKjJjJRrpMxS0DsF4X2NcnqGmTDYOc75+6uK/XN1Uf2d6CzWxiVf34yU8KxWxGibuiIEd7QrhsZurKRm8onlPr5VBncZmjp0bKTJYqj535fmfB0gD5nAxEsVlMVLqnXqul1udgns/BUCzFynofdktpZQYUiplkQnEXQjQKIZ4TQhwQQuwTQnxOP14hhPiDEOKw/r0875q7hRBHhBCHhBDXTucCFNPDke4QS6o9BWPEz19UwVtdIdoDExf1yor7VN0yoGWFHpzgYXJyQIuUMZlKq9WyrlGrM6NcMoq5TjGWewr4WynlCuAi4DNCiJXAXcCzUsplwLP6z+jv3Q6sAq4DHhBCKBNojnGsJ5zrZXoq1+gZn0/v65xwnObeMFUeO55JhkHms3yel2M9YRKp4fDLAx1DfPKh17j+/75IMp2hTQ+DLJX1jZqNojZTFXOdCcVdStkhpXxdfx0EDgDzgVuAh/TTHgJu1V/fAjwipYxLKY8DR4ALDJ63Ai3JqD+coDcUJxRPGTZuJJHiZCA6ZhPoJdUeltZ4+P2+rgnHau6L0FRiQ4rl87ykMjK3D/CDl45zw/0v8sJbvRzoGOKVY305y71UrllVy8aF5WxeUlnyWArFTDIpn7sQYhGwAXgVqJVSdoD2AABq9NPmA615l7Xpx04d604hxA4hxI6enp4pTF3x463NnPeVP7Dpq8+w4X89bVhD6WM9mitlyTh1y69dVcv25n4GwolxxzrRFy65T2i20UbWNfPQ1mY2LijnpS9cictm5rFd7fSG4iVFymRZUu3h0U9fosoMKOY8RYu7EMIDPAr8tZRyPAdoIafnqMpPUsoHpZSbpJSbqquri52GIo+n9nayqNLFl9+5koyEX+5sM2TcsSJl8rl21TzSGcmzB7tHHN97cpD/9dv9bP7aH1nz5d/TNRRn0RQ3U7M0VbmxmgUHO4O
```
Comment (evanyeohboonkhai) [inline notebook/Solution/02 - Autoregressive Integrated Moving Average (ARIMA).ipynb]: please add these comment cause sometime the autoarima may not working correctly
import six
import sys
sys.modules['sklearn.externals.six'] = six
import joblib
sys.modules['sklearn.externals.joblib'] = joblib

**Review by evanyeohboonkhai — COMMENTED** [review]
For AR & MA  model, is good to add the AIC result .
For the stationary

File: notebook/Solution/02 - Stationarity.ipynb [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
...MWb3721nvQ0SfrG6g93NZPvzeBzd27kwXdcxMunuvmBXUuemW71RI9HRLh0eTG7zkSWm2VvYx9blhVFGnwppZQbFlVwf/lUN3/52H5u+5cX+LPt++gasnLivz7cTo8vyD0x1tksyc3iqtWlPHmgNeGN1cGRUZ58o5U7NtfgzUznt7fVs215MV956hgvHO9kY01hwm6LV60u5Uy3n8ZuP72+IKe6fGxZVjTja1ZKqVgWVXB/5NVGCrMz+cg1K/npvhY++p3djIXCfH9nI7VF2Vy3Nnbq420XV9PQ7edgy9SpmZ/vb2VkNMzdW63cuIjwN7dvoMvuJ5NMLfmN9kpCzx7tYK+9IMeWeq1BV0q5a9EE987BAE8dbOPdW+v4q7dt4Mvv3sSuM738zeNv8OKJLu7eVjdp1qjj1o1VpKcJPz/QGvf4xhi+t7ORNRV5XBKVetlcX8Q77UUxtiUR3FeU5bKyLJdnj3awp7GPNCHlVgNKKTXRognuP9jdxGjI8LtXWKmXu7bUcuclNXzf7sR497b6uM8tTiI18/LJbvY39/Ohq1dOyo9/6rb1fOSalUl3XbxhXTkvn+zm5ZNdrKsqiFTvKKWUWxZFcA+HDd97tZE3rSpldXleZPvf3XkRy0pyeMvGqoQLVNy+qZozE1IzgyOjjIyGAPjGb05Snu/hnZdOXru0PN/DX9++IamFNMBKzQTGwrzW0Kv5dqXUrFgUQ8Zv72iguXeYB9564bjthdmZPPUn15GWRCXKLRuq+OufvMHje89yUW0hobDhzq+9RL9/lHdvq+OF41188i0XurI83eUrS8jOTGd4NMSWKaprlFJqphb8yP1/j7TzhZ8f4s0bKnnrRdWTfu7NTCcrI/FlFudmceO6Cn6yt4WxUJjnj3dyqtNHUU4m//6bU+R7MnjvlZOrbWbCm5nO1Wus3jFbpqiLV0qpmVrQI/dDLQN8/Ht72FhTyD/fc0ncG6bJeueldTx1qJ2XTnbzyCuNlOVl8YtPXMfuM71kpAsFXveWqHv/m1aQmZ4Wtw+NUkqlYkEH9/J8a2bnZ9++Mel891RuvLCcwuxMvvHcCXae7uGj168mKyPNlQ6NE11/QbnOSlVKzZqE+QoR+S8R6RCRN6K2lYjI0yJy3P5aHPWzT4nICRE5KiK3ztaJgxXcv/G+ra4tH+fJSOeOzdW8cqoHA7wnxqQnp
```
[the remark this answers is not in the document]
Comment (KianYang-Lee) [reply notebook/Solution/02 - Stationarity.ipynb]: First differncing is not stationary, but second differencing is stationary according to ADF test. Pls advise whether log transfom is still necessary thanks

File: notebook/Solution/02 - AR is not Augmented Reality.ipynb [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
...d3Lpa5dy50d3uvJ4/MVmt7GlfAsACw+AsJt+/9On93ziTQkBTNtzkBvm3uBaf2D5AxTUFvQ8jA681zIKQyEfjZX7v7rfNXGQFJtEalyqa5uzuLKm72jDKILwWdzVOaN21FGW5kgWYKhpqYmIULqkyi7k0UKMVUWrXOE/w5KHcXju4QAIIXjs9MdIiDFqa2wo3cCjKx/l55+5c4tumn8TYzONF6ihyUNdN/z6tvoBDxOoaK5gTBeGUV490GEL2xpTmtDHfM0PS3FM6HR2wt/+Zu14001WKez+YsIES22cYU3QXnzIMkstpYTycpLM9kZamm/vRlISzJtnaTr6kKHM5vz8ta21DKvpIMY8NzJsmCE4YcaHAIOZt3e+zeFPHM6LW17kTyv/xN9WK99lN+yq2kVbZxsAZxenWDf2priuGg63aRN3HHUH0cLwDi47sIxtFdsCZhiFY+kHjZtNZZu4f/n9rvUHTnyAcZluz6mq5qjpPdowihDs0s6B2gPE2WCimnPqw2N0VIR7jIZFgGGUVqvqjYcub+9827V81qSziBLu28u4zHH8euGvXes/+eQnLoWnpNgkfrnwl65tQggPr9FA0WnvpLqx0lXny0Wqe2YuWsLoOi3AoOk/zLO/Q5Md96133oGCAnen+Hi48caBGVBMjEfI2owyd14rGB6JvCpFct9Zb8gXPsLpnF6j0sbSrsPoXIPxrAvkNIzs0s6vP/8157x4DvVt7ommP37zR9o72/EXc37RCQXKq1NvDCNlzLzyCmM6krh0pltB1i7tgfMY6YmcsMVmt3HdW9e5vJzH5B3DLYff4r43oPOMAok2jCKE4oZiOuwdTKzCOruWl+d+qZs716JeNKYORtRHpscobEPplByj7NoOOjo7fHQOLcwy3c4wOjN3Hn0n03IMI90chnb7kbe7Z8UdTMwOrABDR2cHn+77lKrmrpWqqlqqGNIMCeb3u/R0j9ldLcCg6S9aOlpoaDdUxGKjYkmPd9Sge/RRa8fLLhvYumzdhNP1KL/IiWIYOZXpnJLdZU1lXSvSOVF+nzPKjVC62tZaznrhLO778j6PXQrrC3lu03Ndj8+EM78oqxnGHTR58oWARYv8Po6L446D7Gz3emMj3HcfPzn6J5ZuvTKMlGfd0CaoDcPSDxqDP379R9aWrAUgPjqeJ89+kigRpQ2jfkIbRhGCS5HO
```
[the remark this answers is not in the document]
Comment (KianYang-Lee) [reply notebook/Solution/02 - AR is not Augmented Reality.ipynb:471]: done

File: notebook/Solution/02 - AR is not Augmented Reality.ipynb [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+    "The equation for a AR(2) process is: \n",
+    "<br>\n",
+    "<br>\n",
+    "<div style=\"text-align: center;\">$y_t = \\beta_0 + \\beta_1y_{t-1}+\\beta_2y_{t-2}+\\epsilon_t$</div>\n",
+    "<br>\n",
+    "<br>\n",
+    "... and so on. Basically a AR model with higher <i>p</i> order entails the addition of extra lagged terms. A suitable value for <i>p</i> can be determined using PACF plot which was introduced earlier.\n",
+    "\n",
+    "We will be using `statsmodels` module extensively for building MA model. You can read more about the module [here](https://www.statsmodels.org/)."
+   ]
```
[the remark this answers is not in the document]
Comment (KianYang-Lee) [reply notebook/Solution/02 - AR is not Augmented Reality.ipynb:241]: done

File: notebook/Solution/02 - Autoregressive Integrated Moving Average (ARIMA).ipynb (hunk 1 above)
Comment (KianYang-Lee) [reply notebook/Solution/02 - Autoregressive Integrated Moving Average (ARIMA).ipynb]: Thanks BK! Works like a charm. Have updated it

**Comment by evanyeohboonkhai** [comment]
the branch is good to be merged

**Review by evanyeohboonkhai — APPROVED** [review]

**Pull request merged**